packages/turbo/node-platform.js

```diff
@@ -50,7 +50,13 @@ function generateBinPath() {
   // path without modifying the code itself. Do not remove this because
   // external code relies on this (in addition to turbo's own test suite).
   if (TURBO_BINARY_PATH) {
-    return TURBO_BINARY_PATH;
+    if (!fs.existsSync(TURBO_BINARY_PATH)) {
+      console.warn(
+        `[turbo] Ignoring bad configuration: TURBO_BINARY_PATH=${TURBO_BINARY_PATH}`
+      );
```

Comment:
Just confirming, this is lifted from esbuild and not something you ran into during testing, correct?

Reply:
That's correct, but I see the value, so wanted to keep it